webpack/production.js

```diff
@@ -0,0 +1,61 @@
+const path = require('path');
+const webpack = require('webpack');
+
+/**
+ * These are all webpack plugins used to make work
+ * long term caching
+ */
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const ManifestPlugin = require('webpack-manifest-plugin');
+const ChunkManifestPlugin = require('chunk-manifest-webpack-plugin');
+const WebpackMd5Hash = require('webpack-md5-hash');
+const InlineManifestWebpackPlugin = require('inline-manifest-webpack-plugin');
+
+const resolvePath = require('./helpers/resolve-path');
+
+module.exports = {
+  entry: {
+    app: resolvePath('src/index.js')
+  },
+  output: {
+    path: path.join(__dirname, 'build'),
+    filename: '[name].[chunkhash].js',
+    chunkFilename: '[name].[chunkhash].js'
+  },
+  resolve: {
+    extensions: [ '.js', '.jsx' ],
+    modules: [
+      resolvePath('node_modules'),
+      resolvePath('src')
+    ]
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+        query: { cacheDirectory: true }
+      },
+    ],
+  },
+  plugins: [
+    new webpack.optimize.CommonsChunkPlugin({
+      name: 'vendor',
+      chunks: ['app'],
+      minChunks: ({ resource }) => /node_modules/.test(resource)
+    }),
+    new webpack.optimize.CommonsChunkPlugin({
+      name: ['manifest']
+    }),
+    new HtmlWebpackPlugin({
+      template: resolvePath('webpack/html/index.ejs'),
+      env: process.env
+    }),
+    new InlineManifestWebpackPlugin({
+      name: 'webpackManifest'
+    }),
```

Comment:
With `HtmlWebpackPlugin` and `InlineManifestWebpackPlugin` I can see how a manifest is inline in my `index.html`. Is this enough? 
![image](https://cloud.githubusercontent.com/assets/49499/22618828/18a493d6-eae7-11e6-9407-ca0cb795dee7.png)